documentation/Keycloak settings/realm-export-prod.json

```diff
@@ -66,17 +57,17 @@
         "attributes": {}
       },
       {
-        "id": "5f4c16dd-507f-4e8f-b44a-b2a0e188f248",
-        "name": "prime_readonly_admin",
+        "id": "aa118bc3-2b67-4d57-926d-9693412ab437",
+        "name": "prime_administrant",
+        "description": "A role representing the basic ability of the user is to access the \"admin\" portion of PRIME.",
         "composite": false,
         "clientRole": false,
         "containerId": "v4mbqqas",
         "attributes": {}
       },
       {
-        "id": "839e1a7e-31a5-43bd-b303-8e8f6f00bbf6",
-        "name": "feature_site_registration",
-        "description": "Site Registration feature flag",
+        "id": "5f4c16dd-507f-4e8f-b44a-b2a0e188f248",
+        "name": "prime_readonly_admin",
```

Comment:
This role is not referenced in either frontend or backend. Any reason to still keep it in prod?

Reply:
The code currently deployed in prod still references these old roles. We cannot remove them until the new code is deployed all the way into prod.